### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by donaldwasserman**
Got a little too excited there. see #63

### CODE REVIEW

File: tests/dummy/app/templates/components/leaflet-map-example.hbs [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<strong>Notes:</strong>
+<p>
+  Customize the <code>zoom</code> and <code>bounds</code> options by passing a hash of values to <code>zoomPanOptions</code> and <code>fitBounds</code> in your <code>\{{leaflet-map}}</code>.
+  See the <a href="http://leafletjs.com/reference.html#map-fitboundsoptions">avaialble properties here.</a>
```
Comment (miguelcobain) [inline tests/dummy/app/templates/components/leaflet-map-example.hbs]: typo in "avaialble"

File: tests/dummy/app/templates/components/marker-layer-example.hbs [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<strong>Notes</strong>
+<p>
+  To customize your popup, pass a hash of values to the <code>popupOptions</code> property. The <a href="http://leafletjs.com/reference.html#popup">list of available options is available here.</a>
```
Comment (miguelcobain) [inline tests/dummy/app/templates/components/marker-layer-example.hbs]: You can use this link to link to the options directly: http://leafletjs.com/reference.html#popup-options

**Comment by donaldwasserman** [comment]
updated.

File: tests/dummy/app/templates/components/marker-layer-example.hbs [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<strong>Notes</strong>
```
Comment (miguelcobain) [inline tests/dummy/app/templates/components/marker-layer-example.hbs]: Sorry for being picky, but y u no colon here? :)

Comment (donaldwasserman) [inline tests/dummy/app/templates/components/marker-layer-example.hbs]: Lol - I copied it from one of the other examples. 👍 Fixed in both.

**Comment by miguelcobain** [comment]
Thanks a lot for your time!

**Pull request merged**